test/fixtures/experimental/class-private-properties/asi-failure-computed/actual.js

```diff
@@ -1,4 +1,4 @@
 class Foo {
   #p = x
-  [#m] () {}
```

Comment:
This would have been fine actually, besides the method. This looks totally valid to me:
```js
class A {
  #x = 1 // private field
  [#x] = 2 // public field
}
```